baseframe/templates/baseframe/components.html

```diff
@@ -11,6 +11,19 @@
   {% endwith %}
 {%- endmacro %}
 
+{% macro flash_messages_material() -%}
```

Comment:
Let's keep the existing function but add an optional `theme` parameter that defaults to `bootstrap3`.

Reply:
Fixed.  Created a new components template for mui.